components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/export/PhenotipsFamilyExport.java

```diff
@@ -213,12 +214,12 @@ private void queryFamilies(String input, String requiredPermissions, int results
                 continue;
             }
 
-            resultsList.add(new FamilySearchResult(family, requiredPermissions));
+            results.add(new FamilySearchResult(family, requiredPermissions));
         }
     }
 
     private void queryPatients(String input, String requiredPermissions, int resultsLimit,
-        List<FamilySearchResult> resultsList)
+        Set<FamilySearchResult> results)
     {
         StringBuilder querySb = new StringBuilder();
         querySb.append("from doc.object(PhenoTips.PatientClass) as patient, ");
```

Comment:
We can potentially just search for patients whose families are not included in the results from family search.